README.md

```diff
@@ -7,32 +7,14 @@ These images are rebuilt weekly on Thursday nights so if any issues are
 introduced from upstream changes we have the opportunity to resolve them before
 the weekend.
 
-## A note on `php-apache-nvm`
+## A note on `php-apache-n`
 
-The `php-apache-nvm` image installs nvm and creates a script to easily install
-your project’s required version of Node.js. Running the following from a
-project that uses this Docker image will install the version of Node specified
-in your project’s `.nvmrc` file and then install Yarn:
+The `php-apache-n` image installs [n](https://github.com/tj/n) and installs
```

Comment:
This readme section cleared up so much for me. I was struggling to wrap my head around the big picture. 🏆